tests/evaluator-tests.js

```diff
@@ -21,44 +21,44 @@ describe('Evaluator', () => {
       arr: ['zero', 'one']
     })
 
-    it("return entire map when passed `[]`", () => {
+    it('return entire map when passed `[]`', () => {
       var result = evaluator.evaluate(state, [])
       expect(Immutable.is(state, result)).toBe(true)
     })
 
-    it("['top']", () => {
+    it('[\'top\']', () => {
```

Comment:
@jordangarcia I like enforcing consistency over escaping. I think with eslint you can do `[2, "single", "avoid-escape"]` which would result in:

var double = "double" //exception
var single = 'single' //all good
var noEscape = '<div class="no-escaping">' //all good